index.js

```diff
@@ -0,0 +1,88 @@
+/// DOM Elements ///
+const playerDiv = document.querySelector('div.player')
+const playerImg = document.querySelector('img')
+const playerName = document.querySelector('h2')
+const playerNickName = document.querySelector('em')
+const playerLikes = document.querySelector('p.likes')
+const playerGoals = document.querySelector('ul#goals')
+
+const playerGoalForm = document.querySelector('#new-goal-form')
+
+
+const likeBtn = document.querySelector('.like-button')
+
+// console.log(likeBtn)
+/////////////////////// Deliverable 1 /////////////////////////////
+function intialize() {
+    fetch('http://localhost:3000/players/1')
+    .then(response => response.json())
+    .then(playerData => {
+        playerImg.src = playerData.photo,
+        playerName.textContent = playerData.name,
+        playerNickName.textContent = playerData.nickname,
+        playerLikes.textContent = `${playerData.likes} Likes`,
+        playerData.goals.forEach(goal => renderGoals(goal))
```

Comment:
It might be worth creating a separate function for this, just for readability and clean code.